### ISSUE TITLE
How to Fine Tune a model in Turicreate?

### ISSUE THREAD

**Issue by ShreshthSaxena** [apple/turicreate#2563]
How can I fine tune a model such as those available here : https://developer.apple.com/machine-learning/models/ 

I need to detect one class of basketballs by training on a small dataset 
Also I cannot find a way to change learning rate, freeze layers and modify parameters in TuriCreate. Is it not possible anyhow or already taken care of by tc?

**Comment by TobyRoseman** [comment]
Hi @ShreshthSaxena - In general TuriCreate tries to do a lot of implementation details (such as setting learning rate and freezing layers) for you.

It sounds like you're interested in either [Image Classification](https://apple.github.io/turicreate/docs/userguide/image_classifier/) or [Object detection](https://apple.github.io/turicreate/docs/userguide/object_detection/). If that's not the case, please let us know. Also let us know if you have any other questions.

**Comment by ShreshthSaxena** [comment]
I'm interested in Object Detection and I've gone through the user guides already. I am looking to modify the pretrained models provided in https://developer.apple.com/machine-learning/models/
and fine tune them for my use. 
For instance how can I alter the Yolov3 model to only detect one object (basketball in my case)
Or how can I change the base model from Yolov2 in tc.object_detector.create() ?

**Comment by TobyRoseman** [comment]
Currently the only model supported for object detection is `darknet-yolo`. We're actively working on adding more models. We are already tracking that issue in #1734.

It sounds like your use case is a little different. You want to use an arbitrary object detection model as a starting point for a TuriCreate object detection model. This is also currently not supported. I'll leave this issue open as a feature request.

**Comment by ShreshthSaxena** [comment]
Okay, Thanks
When should we expect more backbones other than darkness-yolo for Object Detection?
